en/sql/catalog.rst

```diff
@@ -473,6 +473,8 @@ Represents index information. An index for class_of is created.
 +--------------------+---------------------------+------------------------------------------------+
 | comment            | VARCHAR (1024)            | Comment to describe the index                  |
 +--------------------+---------------------------+------------------------------------------------+
+| status             | INTEGER                   | Index status                  |
```

Comment:
please make sure the this row in table if properly formatted (in github view it appears it requires more spaces)